<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackagesManager</class>
 <widget class="QMainWindow" name="PackagesManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Packages Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="enableButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disableButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Package</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Version</string>
    </property>
   </widget>
   <widget class="QListWidget" name="versionField">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>111</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="packageField">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>111</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QListWidget" name="disableField">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>111</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disabled</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
